libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -644,6 +626,7 @@ void AC_AttitudeControl::input_thrust_vector_heading(const Vector3f& thrust_vect
     } else {
         // set persisted quaternion target attitude
         _attitude_target = desired_attitude_quat;
+        _attitude_target.to_euler(_euler_angle_target);
```

Comment:
Why are you adding this here?

Reply:
`_euler_angle_target` used in `ang_vel_to_euler_rate` on 637 before its updated in `attitude_controller_run_quat`

Reply:
In this case _ang_vel_target.zero();